studio/src/window.cpp

```diff
@@ -733,6 +733,10 @@ void Window::onExport(bool)
     {
         return;
     }
+    if (!export_filename.endsWith(".stl"))
```

Comment:
Want to make this check against the lowercase filename, in case someone like `.STL`?  That should be just
`export_filename.toLower().endsWith(".stl")`.